share/doc/DEVELOPERS_GUIDE.rst

```diff
@@ -468,9 +475,11 @@ Updating Git Submodules
 Votca with all of its repos can be build by using the parent `votca
```

Comment:
@JensWehner this whole section needs a do over as the @votca-bot does all these update automatically now.